src/span-processor-builder.ts

```diff
@@ -18,12 +19,15 @@ import { configureHoneycombHttpJsonTraceExporter } from './http-json-trace-expor
 export const configureSpanProcessors = (options?: HoneycombOptions) => {
   const honeycombSpanProcessor = new CompositeSpanProcessor();
 
-  // We have to configure the exporter here becuase the way the base SDK is setup
+  // We have to configure the exporter here because the way the base SDK is setup
   // does not allow having both a `spanProcessor` and `traceExporter` configured.
   honeycombSpanProcessor.addProcessor(
     new BatchSpanProcessor(configureHoneycombHttpJsonTraceExporter(options)),
   );
 
+  // we always want to add the baggage span processor
```

Comment:
consider for later: make this configurable? It's not configurable in the other distros but we've talked about it in case someone doesn't want it.

Reply:
do you think it'd be useful to add the session id to the baggage? thinking it would help link server-side traces to specific sessions

Reply:
Ooh adding session id sounds like a great idea! Will set a follow-up for that.

Reply:
- #75